pkg/controller/compute/gke.go

```diff
@@ -66,6 +76,7 @@ var (
 type Reconciler struct {
 	client.Client
 	publisher resource.ManagedConnectionPublisher
+	resource.ManagedReferenceResolver
```

Comment:
I'd prefer this wasn't embedded - it's a little strange to have a non embedded `resource.ManagedConnectionPublisher` followed immediately by an embedded `resource.ManagedReferenceResolver`.

Reply:
fixed